pseudo_code/linear_fit.py

```diff
@@ -0,0 +1,54 @@
+def linear_fit(data):
```

Comment:
What type of data is expected in this input? This is not a request for a change, more elaboration, since if it is expected to be a csv file, the code will need to be adjusted a bit.